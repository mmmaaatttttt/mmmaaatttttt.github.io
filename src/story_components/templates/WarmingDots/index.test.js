import React from "react";
import { render } from "@testing-library/react";
import { WarmingDots } from ".";

describe("smoke and snapshot tests", () => {
  it("renders successfully", () => {
    render(<WarmingDots />);
  });

  it("renders first demonstration successfully", () => {
    const { asFragment } = render(<WarmingDots idx={0} />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("renders second demonstration successfully", () => {
    const { asFragment } = render(<WarmingDots idx={1} />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("renders third demonstration successfully", () => {
    const { asFragment } = render(<WarmingDots idx={2} />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("renders fourth demonstration successfully", () => {
    const { asFragment } = render(<WarmingDots idx={3} />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("renders fifth demonstration successfully", () => {
    const { asFragment } = render(<WarmingDots idx={4} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
