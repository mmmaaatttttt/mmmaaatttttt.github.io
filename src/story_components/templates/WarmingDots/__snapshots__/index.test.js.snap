// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`smoke and snapshot tests renders fifth demonstration successfully 1`] = `
<SliderProvider
  column={true}
  fullWidthAt="small"
  initialData={
    Array [
      Object {
        "color": "#5ECFFF",
        "equationParameter": true,
        "initialValue": 1,
        "max": 5,
        "maxIcon": "fast-forward",
        "min": 0,
        "minIcon": "pause",
        "title": "Growth rate for the population",
      },
    ]
  }
  render={[Function]}
  width="70%"
/>
`;

exports[`smoke and snapshot tests renders first demonstration successfully 1`] = `
<SliderProvider
  column={true}
  fullWidthAt="small"
  initialData={
    Array [
      Object {
        "color": "#5ECFFF",
        "equationParameter": true,
        "initialValue": 1,
        "max": 5,
        "maxIcon": "fast-forward",
        "min": 0,
        "minIcon": "pause",
        "title": "Growth rate for the population",
      },
    ]
  }
  render={[Function]}
  width="70%"
/>
`;

exports[`smoke and snapshot tests renders fourth demonstration successfully 1`] = `
<SliderProvider
  column={true}
  fullWidthAt="small"
  initialData={
    Array [
      Object {
        "color": "#5ECFFF",
        "equationParameter": true,
        "initialValue": 1,
        "max": 5,
        "maxIcon": "fast-forward",
        "min": 0,
        "minIcon": "pause",
        "title": "Growth rate for the population",
      },
    ]
  }
  render={[Function]}
  width="70%"
/>
`;

exports[`smoke and snapshot tests renders second demonstration successfully 1`] = `
<SliderProvider
  column={true}
  fullWidthAt="small"
  initialData={
    Array [
      Object {
        "color": "#5ECFFF",
        "equationParameter": true,
        "initialValue": 1,
        "max": 5,
        "maxIcon": "fast-forward",
        "min": 0,
        "minIcon": "pause",
        "title": "Growth rate for the population",
      },
    ]
  }
  render={[Function]}
  width="70%"
/>
`;

exports[`smoke and snapshot tests renders third demonstration successfully 1`] = `
<SliderProvider
  column={true}
  fullWidthAt="small"
  initialData={
    Array [
      Object {
        "color": "#5ECFFF",
        "equationParameter": true,
        "initialValue": 1,
        "max": 5,
        "maxIcon": "fast-forward",
        "min": 0,
        "minIcon": "pause",
        "title": "Growth rate for the population",
      },
    ]
  }
  render={[Function]}
  width="70%"
/>
`;
